We always want to install configs as the host machine may be different from
the one where zmap is installed to. Plus we have CONFIG_PROTECT in Gentoo
for protecting the user's existing configuration.

Bug: https://bugs.gentoo.org/885707
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -135,11 +135,7 @@ message(STATUS "Default ZMap configuration file location is /etc/zmap")
 foreach(EACH_CONF ${CONF_FILES})
     get_filename_component(CONF_BASENAME ${EACH_CONF} NAME)
     message(STATUS "Checking if ${CONF_BASENAME} exists there...")
-    if(NOT EXISTS "/etc/zmap/${CONF_BASENAME}")
-        install(FILES ${EACH_CONF} DESTINATION ${CONFIG_DESTINATION})
-    else()
-        message(WARNING "Existing configuration file detected at /etc/zmap/${CONF_BASENAME}, ${CONF_BASENAME} from sources will NOT be installed. Please check and install manually!")
-    endif()
+    install(FILES ${EACH_CONF} DESTINATION ${CONFIG_DESTINATION})
 endforeach()
 
 # Allow Debian Packaging
